admin/spec/controllers/restart.spec.js

```diff
@@ -0,0 +1,81 @@
+'use strict'
+/* global describe beforeEach afterEach it expect jasmine spyOn */
+const proxyquire = require('proxyquire')
+const httpMocks = require('node-mocks-http')
+const sinon = require('sinon')
```

Comment:
I wouldn't expect to see this in a controller.  Is it used?

Reply:
@jon-shipley sinon and proxyquire are not used.
Updated